### PR TITLE
Wooc 763

### PR DESCRIPTION
**Pull request by nunoteixeiranat**

### CODE REVIEW

File: src/Controller/ApplePay.php [hunk 1]
```diff
@@ -164,9 +164,11 @@ public function add_apple_pay_js() {
 			array(
 				'ajax_url_payplug_create_order' => \WC_AJAX::get_endpoint('payplug_create_order'),
 				'ajax_url_applepay_update_payment' => \WC_AJAX::get_endpoint('applepay_update_payment'),
+				'ajax_url_applepay_get_order_totals' => \WC_AJAX::get_endpoint('applepay_get_order_totals'),
 				'countryCode' => WC()->customer->get_billing_country(),
 				'currencyCode' => get_woocommerce_currency(),
-				'total' => WC()->cart->total,
+				'total'  => WC()->cart->total,
+				//'shipping_costs' => WC()->cart->get_shipping_total(),
```
Comment (PPmmesquita) [inline src/Controller/ApplePay.php:171]: remove the comment line

File: src/Controller/ApplePay.php [hunk 2]
```diff
@@ -164,9 +164,11 @@ public function add_apple_pay_js() {
 			array(
 				'ajax_url_payplug_create_order' => \WC_AJAX::get_endpoint('payplug_create_order'),
 				'ajax_url_applepay_update_payment' => \WC_AJAX::get_endpoint('applepay_update_payment'),
+				'ajax_url_applepay_get_order_totals' => \WC_AJAX::get_endpoint('applepay_get_order_totals'),
 				'countryCode' => WC()->customer->get_billing_country(),
 				'currencyCode' => get_woocommerce_currency(),
-				'total' => WC()->cart->total,
+				'total'  => WC()->cart->total,
```
Comment (PPmmesquita) [inline src/Controller/ApplePay.php:170]: are you using the `'total'`?

File: assets/js/payplug-apple-pay.js [hunk 3]
```diff
@@ -121,6 +122,24 @@
 					},
 				})
 			}
+		},
+
+		getOrderTotals(){
+			if(!apple_pay.load_order_total){
```
Comment (PPmmesquita) [inline assets/js/payplug-apple-pay.js:128]: this is for the first time the pages load to not make the call right?
Can you put a little comment with this info?

**Review by PPmmesquita — CHANGES_REQUESTED** [review]

File: src/Controller/ApplePay.php (hunk 2 above)
Comment (nunoteixeiranat) [reply src/Controller/ApplePay.php:170]: yes

**Pull request merged**